core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -205,6 +205,8 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
   def toValidatedNel(implicit F: Functor[F]): F[ValidatedNel[A, B]] =
     F.map(value)(_.toValidatedNel)
 
+  def toValidatedNec(implicit F: Functor[F]): F[ValidatedNec[A, B]] = ???
```

Comment:
implementation coming, I assume....

Reply:
this is still a `???`

We should consider a CI check that we can't merge with that present.

Reply:
Also, can we add a test to exercise this, since we must not have it now.

Reply:
@johnynek I think that you are looking at outdated code. For me there's no longer `???` in the PR.